src/lib/interfaces/digital-asset-deployment.ts

```diff
@@ -28,6 +28,11 @@ export interface LSP7DigitalAssetDeploymentOptions extends DigitalAssetDeploymen
   isNFT: boolean;
 }
 
+export interface LSP8IdentifiableDigitalAssetDeploymentOptions
+  extends DigitalAssetDeploymentOptions {
+  tokenIdType: 1 | 2 | 3 | 4 | 5;
```

Comment:
can we use an imported type from lsp-smart contracts?

Reply:
There is the constant in the lsp-smart-contracts (the one I use in the tests), but I don't think it has a type.

LSP8 define the types from 0 to 4 (so it's actually wrong what I wrote here).

But technically, anyone defining a custom tokenIdType (e.g: 8) should be allowed to do so.
So we can leave it to number.

Reply:
okay but in that case we will only allow custom ids 5,6,7,8. Any other will result in a type error

Reply:
ah I see you mean change it to a number type? That makes sense. Could it ever be a string?